scripts/curate_basel.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+import json
+import argparse
+
+root_data = '/Users/alex/data/basel-mp2rage_raw'
+output_data = '/Users/alex/data/basel-mp2rage_raw_curated'
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(root_data, output_data):
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data)
+
+    dict_images = {
+        "MP2RAGE_UNI_Images.nii.gz": "_UNIT1.nii.gz"
+    }
+
+    dict_der = {
+        "lesion_mask_sc.nii.gz": "_UNIT1_lesion-manual.nii.gz"
+    }
+
+    for dirs, subdirs, files in os.walk(root_data):
+        for file in files:
+            if file.endswith('.nii.gz') and file in dict_images.keys() or file in dict_der.keys():
+                path_file_in = os.path.join(dirs, file)
+                path = os.path.normpath(path_file_in)
+                subid_bids = 'sub-' + (path.split(os.sep))[5]
```

Comment:
What's the `[5]` for? Is that tied to ` root_data = '/Users/alex/data/basel-mp2rage_raw'`? So `[5] would be the filename within that folder?

You can use `os.path.basename(path)` for that. It'll do the right thing.

Reply:
I need the folder where the `path_file_in` is . For example, for `/Users/alex/data/basel-mp2rage_raw/P027/lesion_mask_sc.nii.gz` I need `P027`.

The suggestion `os.path.basename(path)` doesn't seem to do it.